.github/workflows/pr_style_check.yaml

```diff
@@ -38,6 +38,12 @@ jobs:
             echo "The first line of the PR description must be '**Commit Message**'"
             exit 1
           fi
+      - name: Check no markdown format link in '**Commit Message**'
+        run: |
+          if [[ echo "$input" | grep -q '\[.*\]\(.*\)']]; then
```

Comment:
obviously this is not working. where's $input?

Reply:
Sorry I use input to test.